packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -5,6 +5,8 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
+const POPOVER_HANDLE_WIDTH = 25;
```

Comment:
Rename is doable, comments are doable... I'm just not sure how useful comments will be given that they're not going to be surfaced in any way anyone will be able to see.